src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -167,7 +169,7 @@ function OptionRowLHN(props) {
                             if (e) {
                                 e.preventDefault();
                             }
-
+                            ReportActionComposeFocusManager.focus();
```

Comment:
Could you add an explanatory comment here? I don't think it will be immediately clear for future contributors what we're doing here - same for the other line.